test/test_vte_mouse.c

```diff
@@ -27,6 +27,8 @@
 #include "libtsm.h"
 #include "libtsm-int.h"
 
+#define ESC "\x1b"
```

Comment:
Why can't we directly use "\x1b"? I feel using a macro here doesn't add too much value.